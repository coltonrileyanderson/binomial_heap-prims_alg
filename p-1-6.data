47 58 64 ; 36 32 376 ; 33 2 416 ; 45 23 202 ; 19 31 150 ; 
46 36 221 ; 6 13 5 ; 7 49 493 ; 22 7 422 ; 62 20 227 ; 
56 17 14 ; 24 13 197 ; 26 37 210 ; 49 52 83 ; 17 14 480 ; 
53 39 131 ; 3 1 297 ; 52 18 183 ; 59 33 338 ; 16 60 100 ; 
33 11 250 ; 46 8 218 ; 20 42 130 ; 8 45 196 ; 22 29 126 ; 
62 32 439 ; 63 17 332 ; 17 8 403 ; 50 6 279 ; 47 27 71 ; 
58 45 47 ; 2 56 124 ; 45 26 326 ; 26 23 422 ; 56 5 466 ; 
24 4 109 ; 42 56 423 ; 50 48 62 ; 57 27 371 ; 20 2 478 ; 
1 57 257 ; 57 17 427 ; 20 24 185 ; 43 50 94 ; 48 21 89 ; 
52 14 149 ; 44 5 257 ; 28 59 63 ; 56 2 398 ; 58 60 451 ; 
52 37 175 ; 5 8 320 ; 29 46 106 ; 16 0 14 ; 37 45 389 ; 
51 18 209 ; 56 16 265 ; 51 55 389 ; 53 17 338 ; 50 36 38 ; 
23 11 264 ; 32 52 212 ; 14 22 210 ; 14 12 83 ; 60 45 255 ; 
14 62 192 ; 30 3 207 ; 22 52 371 ; 39 2 246 ; 12 8 56 ; 
23 9 482 ; 11 61 77 ; 33 55 473 ; 45 15 355 ; 26 54 170 ; 
24 3 6 ; 27 2 364 ; 16 58 151 ; 18 36 133 ; 27 4 356 ; 
36 22 124 ; 33 0 316 ; 25 9 251 ; 40 33 389 ; 30 7 390 ; 
33 41 349 ; 36 25 121 ; 30 40 325 ; 2 13 53 ; 7 26 291 ; 
29 50 282 ; 29 34 438 ; 39 30 176 ; 8 59 205 ; 15 5 109 ; 
56 42 495 ; 17 40 261 ; 58 1 409 ; 7 42 15 ; 33 37 473 ; 
20 60 396 ; 54 54 62 ; 20 18 251 ; 15 7 86 ; 21 36 478 ; 
63 32 245 ; 39 61 97 ; 40 62 47 ; 18 29 449 ; 56 62 84 ; 
52 39 162 ; 42 45 158 ; 61 16 388 ; 4 27 457 ; 41 45 252 ; 
9 15 369 ; 6 40 305 ; 4 19 180 ; 33 56 25 ; 31 37 391 ; 
6 51 423 ; 51 21 403 ; 4 45 326 ; 31 26 163 ; 12 23 323 ; 
27 40 416 ; 4 57 289 ; 23 50 55 ; 16 19 123 ; 53 6 156 ; 
40 50 480 ; 61 17 194 ; 42 27 10 ; 5 34 178 ; 28 46 489 ; 
4 34 69 ; 62 32 17 ; 48 35 281 ; 3 43 112 ; 9 19 256 ; 
59 27 1 ; 13 42 9 ; 40 44 290 ; 11 50 29 ; 57 23 238 ; 
27 8 170 ; 59 21 194 ; 31 38 116 ; 10 15 468 ; 29 7 275 ; 
57 53 429 ; 35 10 204 ; 15 29 357 ; 1 26 214 ; 24 28 289 ; 
32 58 202 ; 54 28 425 ; 28 61 249 ; 43 13 80 ; 51 33 190 ; 
40 62 87 ; 50 22 104 ; 15 60 435 ; 41 10 90 ; 5 35 99 ; 
63 53 99 ; 28 34 12 ; 25 11 354 ; 38 19 110 ; 7 37 42 ; 
5 21 257 ; 28 61 257 ; 24 9 403 ; 34 35 487 ; 5 63 388 ; 
58 50 198 ; 28 6 85 ; 40 16 245 ; 59 55 294 ; 33 9 53 ; 
54 14 444 ; 51 59 250 ; 60 26 26 ; 32 9 482 ; 31 34 480 ; 
17 0 365 ; 24 29 135 ; 40 23 419 ; 31 41 319 ; 40 34 63 ; 
54 35 44 ; 49 41 350 ; 12 11 85 ; 21 5 390 ; 55 33 206 ; 
55 2 281 ; 21 29 66 ; 44 20 74 ; 21 52 159 ; 56 15 282 ; 
27 11 135 ; 4 61 325 ; 15 34 463 ; 20 4 451 ; 53 6 128 ; 
55 39 236 ; 20 26 471 ; 40 36 124 ; 28 23 440 ; 43 42 214 ; 
54 37 350 ; 51 22 145 ; 22 57 201 ; 26 40 44 ; 33 45 8 ; 
8 41 425 ; 34 28 308 ; 7 37 144 ; 30 25 204 ; 8 35 161 ; 
45 37 76 ; 4 2 495 ; 61 35 470 ; 38 19 88 ; 19 23 239 ; 
60 16 332 ; 24 0 296 ; 61 4 255 ; 23 49 430 ; 58 58 480 ; 
32 46 393 ; 34 25 302 ; 6 56 34 ; 25 44 200 ; 49 59 306 ; 
1 43 425 ; 2 36 349 ; 6 63 197 ; 55 33 95 ; 49 33 186 ; 
32 42 184 ; 57 33 440 ; 49 27 451 ; 30 19 65 ; 25 49 30 ; 
5 34 3 ; 41 15 261 ; 40 21 463 ; 9 24 105 ; 42 50 41 ; 
21 6 134 ; 54 59 110 ; 17 18 168 ; 37 47 277 ; 22 37 27 ; 
57 13 9 ; 8 54 406 ; 29 33 163 ; 53 18 387 ; 39 59 8 ; 
46 22 210 ; 41 16 387 ; 59 4 341 ; 42 42 459 ; 15 45 440 ; 
28 10 90 ; 18 44 39 ; 51 55 284 ; 6 42 373 ; 1 38 255 ; 
23 13 262 ; 40 8 136 ; 44 58 465 ; 22 61 52 ; 4 37 462 ; 
14 35 226 ; 59 27 198 ; 50 27 493 ; 29 56 220 ; 3 11 42 ; 
16 6 296 ; 25 46 467 ; 19 0 167 ; 16 12 416 ; 53 3 490 ; 
24 9 418 ; 51 43 413 ; 14 32 380 ; 7 9 482 ; 18 55 107 ; 
24 39 467 ; 7 47 271 ; 7 30 408 ; 19 58 219 ; 22 45 163 ; 
31 35 384 ; 11 40 269 ; 43 40 412 ; 52 27 99 ; 44 42 54 ; 
19 61 32 ; 2 33 297 ; 33 29 484 ; 27 45 263 ; 8 12 334 ; 
44 33 161 ; 20 14 320 ; 61 46 363 ; 24 29 283 ; 2 19 299 ; 
0 28 159 ; 33 61 307 ; 62 37 61 ; 43 20 90 ; 56 22 162 ; 
25 47 118 ; 39 40 230 ; 21 43 59 ; 51 26 188 ; 6 62 452 ; 
34 13 458 ; 31 48 26 ; 4 22 427 ; 24 28 70 ; 47 62 25 ; 
30 41 309 ; 6 63 306 ; 50 42 285 ; 12 18 11 ; 10 18 82 ; 
24 21 404 ; 8 23 11 ; 30 59 50 ; 59 29 155 ; 57 42 439 ; 
34 48 90 ; 34 14 417 ; 12 20 443 ; 31 47 181 ; 49 61 266 ; 
6 17 200 ; 30 8 292 ; 25 31 201 ; 54 58 339 ; 32 45 150 ; 
16 16 444 ; 31 17 394 ; 51 24 380 ; 34 18 461 ; 31 24 28 ; 
48 40 33 ; 57 17 288 ; 24 2 318 ; 18 17 51 ; 63 28 409 ; 
15 32 417 ; 32 6 90 ; 57 15 381 ; 11 36 428 ; 35 24 242 ; 
11 61 405 ; 28 41 25 ; 30 51 262 ; 48 22 18 ; 12 47 300 ; 
44 41 196 ; 50 51 499 ; 1 28 424 ; 37 56 469 ; 61 54 112 ; 
59 19 323 ; 36 32 154 ; 24 39 47 ; 46 5 53 ; 30 54 217 ; 
7 38 337 ; 58 37 61 ; 22 5 284 ; 14 30 385 ; 4 2 179 ; 
23 9 192 ; 55 28 163 ; 30 11 176 ; 36 24 104 ; 26 22 442 ; 
0 31 156 ; 37 21 163 ; 43 61 201 ; 9 38 355 ; 12 53 405 ; 
21 2 445 ; 49 9 232 ; 60 53 440 ; 21 57 182 ; 43 44 478 ; 
10 60 332 ; 31 20 88 ; 28 37 450 ; 3 13 489 ; 56 2 26 ; 
58 60 174 ; 4 12 108 ; 57 20 333 ; 51 59 398 ; 31 5 355 ; 
27 42 422 ; 47 14 288 ; 21 24 368 ; 34 13 444 ; 36 12 51 ; 
32 22 220 ; 45 54 346 ; 1 8 242 ; 60 22 288 ; 2 57 235 ; 
44 7 378 ; 58 31 77 ; 19 23 293 ; 32 44 430 ; 44 3 344 ; 
3 19 292 ; 57 30 73 ; 1 44 66 ; 23 52 63 ; 16 47 396 ; 
23 9 246 ; 55 63 446 ; 14 36 84 ; 58 30 315 ; 62 58 276 ; 
17 58 492 ; 47 27 343 ; 28 41 85 ; 16 57 254 ; 0 5 46 ; 
9 19 370 ; 8 13 446 ; 44 1 265 ; 10 9 47 ; 4 49 238 ; 
63 5 36 ; 26 29 217 ; 4 46 425 ; 61 0 309 ; 2 42 453 ; 
21 37 445 ; 34 55 367 ; 35 0 295 ; 44 23 24 ; 29 41 186 ; 
35 57 127 ; 0 27 490 ; 46 44 348 ; 46 25 120 ; 25 22 17 ; 
62 20 139 ; 53 27 232 ; 53 42 242 ; 12 38 106 ; 54 45 138 ; 
47 56 116 ; 10 2 383 ; 54 38 135 ; 15 15 457 ; 38 52 103 ; 
58 28 324 ; 21 24 138 ; 63 55 195 ; 38 57 408 ; 19 39 381 ; 
11 51 147 ; 13 34 446 ; 51 17 349 ; 2 46 427 ; 55 58 343 ; 
19 30 401 ; 43 1 195 ; 34 24 296 ; 27 37 147 ; 3 62 477 ; 
54 13 324 ; 25 11 463 ; 42 20 88 ; 25 20 400 ; 19 35 166 ; 
49 24 257 ; 50 7 16 ; 11 11 28 ; 48 14 182 ; 46 56 290 ; 
59 52 489 ; 6 60 247 ; 27 16 448 ; 47 24 431 ; 18 35 152 ; 
42 36 67 ; 49 57 0 ; 60 57 295 ; 10 27 294 ; 2 25 258 ; 
55 26 420 ; 51 18 279 ; 3 34 281 ; 28 29 276 ; 63 49 395 ; 
35 24 129 ; 28 49 83 ; 21 60 304 ; 49 18 236 ; 10 24 379 ; 
37 12 265 ; 55 59 16 ; 25 48 447 ; 55 1 499 ; 40 0 71 ; 
1 18 182 ; 50 13 89 ; 46 58 162 ; 0 34 279 ; 24 21 272 ; 
36 11 135 ; 31 55 356 ; 15 23 331 ; 16 58 212 ; 17 57 283 ; 
35 55 196 ; 48 24 369 ; 42 22 7 ; 13 42 53 ; 34 46 153 ; 
45 61 417 ; 37 49 117 ; 60 28 180 ; 54 33 91 ; 47 28 480 ; 
38 0 389 ; 62 37 300 ; 21 60 0 ; 63 1 443 ; 45 6 216 ; 
43 35 198 ; 28 36 8 ; 57 4 161 ; 26 52 253 ; 54 52 487 ; 
55 5 286 ; 28 57 480 ; 25 29 10 ; 26 44 243 ; 33 8 106 ; 
4 7 76 ; 41 51 435 ; 45 4 243 ; 33 17 332 ; 21 20 266 ; 
26 6 477 ; 19 30 12 ; 49 52 92 ; 29 32 373 ; 38 10 260 ; 
45 34 109 ; 32 37 485 ; 36 5 6 ; 53 29 56 ; 9 16 225 ; 
15 57 495 ; 45 25 105 ; 33 21 5 ; 1 43 398 ; 12 27 227 ; 
46 44 209 ; 20 1 246 ; 25 0 346 ; 54 44 170 ; 6 57 323 ; 
63 28 292 ; 24 26 363 ; 46 43 220 ; 25 45 22 ; 52 12 215 ; 
32 18 407 ; 33 4 378 ; 33 23 210 ; 13 13 499 ; 7 57 281 ; 
35 45 324 ; 61 49 103 ; 40 33 484 ; 21 51 206 ; 34 7 347 ; 
52 38 167 ; 56 52 423 ; 15 22 304 ; 29 17 318 ; 22 3 371 ; 
3 59 190 ; 52 50 122 ; 22 2 123 ; 9 0 200 ; 16 23 249 ; 
55 34 351 ; 43 13 215 ; 2 26 292 ; 19 8 427 ; 22 3 496 ; 
17 6 118 ; 4 44 290 ; 6 15 226 ; 7 27 148 ; 30 61 78 ; 
1 0 82 ; 14 56 421 ; 40 40 121 ; 48 39 95 ; 52 4 342 ; 
58 54 145 ; 38 44 326 ; 53 23 194 ; 17 23 381 ; 14 33 92 ; 
14 60 378 ; 6 13 203 ; 46 10 46 ; 22 9 210 ; 26 28 219 ; 
16 5 239 ; 61 63 165 ; 20 62 119 ; 43 3 224 ; 13 15 385 ; 
9 57 147 ; 22 37 66 ; 33 31 208 ; 42 1 367 ; 6 20 469 ; 
11 27 363 ; 11 28 371 ; 9 3 80 ; 12 32 124 ; 27 13 96 ; 
20 9 277 ; 57 19 250 ; 24 35 436 ; 26 34 309 ; 46 55 464 ; 
10 38 94 ; 38 38 198 ; 42 34 101 ; 10 51 350 ; 23 47 178 ; 
32 56 177 ; 51 10 225 ; 23 50 231 ; 57 60 387 ; 49 48 461 ; 
23 42 468 ; 61 48 350 ; 32 1 124 ; 19 27 58 ; 2 1 228 ; 
58 2 372 ; 4 24 284 ; 54 39 88 ; 50 14 287 ; 34 16 452 ; 
13 0 201 ; 61 58 249 ; 63 26 318 ; 26 37 490 ; 28 57 241 ; 
30 37 315 ; 54 50 193 ; 29 50 477 ; 44 9 456 ; 60 25 86 ; 
60 27 185 ; 54 52 198 ; 16 10 471 ; 53 1 310 ; 47 10 65 ; 
20 22 265 ; 6 27 87 ; 56 37 110 ; 1 37 415 ; 27 23 39 ; 
54 44 147 ; 42 14 355 ; 52 46 217 ; 53 12 6 ; 0 5 483 ; 
22 26 26 ; 49 14 359 ; 22 56 265 ; 59 63 186 ; 19 51 259 ; 
63 34 86 ; 14 14 72 ; 60 47 114 ; 8 33 235 ; 14 9 424 ; 
40 31 424 ; 54 10 254 ; 46 60 256 ; 46 38 462 ; 33 17 330 ; 
3 24 445 ; 17 20 344 ; 1 59 78 ; 34 22 118 ; 43 38 479 ; 
11 55 21 ; 21 29 213 ; 17 53 144 ; 55 36 422 ; 9 14 112 ; 
33 11 309 ; 53 7 309 ; 48 18 170 ; 6 24 259 ; 45 39 434 ; 
36 4 29 ; 1 10 72 ; 55 6 422 ; 27 48 195 ; 41 9 79 ; 
52 2 332 ; 59 63 393 ; 13 14 38 ; 38 29 31 ; 13 15 105 ; 
18 45 38 ; 28 21 254 ; 34 19 8 ; 18 35 407 ; 27 42 266 ; 
30 58 415 ; 29 43 299 ; 43 17 466 ; 9 40 496 ; 24 6 6 ; 
5 16 179 ; 27 63 185 ; 46 47 293 ; 17 22 295 ; 59 24 466 ; 
53 3 269 ; 32 34 163 ; 50 60 13 ; 26 36 284 ; 32 50 293 ; 
49 14 270 ; 48 59 95 ; 31 5 46 ; 53 48 45 ; 13 46 434 ; 
16 8 498 ; 50 63 215 ; 46 20 332 ; 19 21 238 ; 5 30 112 ; 
19 0 265 ; 15 40 234 ; 20 42 126 ; 4 27 97 ; 51 30 110 ; 
59 28 227 ; 58 15 67 ; 15 35 429 ; 36 41 279 ; 2 10 114 ; 
3 11 144 ; 43 61 343 ; 21 31 306 ; 49 32 122 ; 15 22 291 ; 
43 25 130 ; 58 44 409 ; 30 49 495 ; 7 36 198 ; 17 43 358 ; 
28 55 417 ; 25 22 144 ; 56 5 339 ; 24 7 48 ; 46 14 317 ; 
7 8 80 ; 51 61 39 ; 36 45 58 ; 9 43 320 ; 52 14 415 ; 
43 51 105 ; 1 27 209 ; 6 46 111 ; 14 22 287 ; 28 48 336 ; 
37 24 70 ; 34 44 366 ; 15 58 493 ; 59 38 30 ; 9 25 107 ; 
60 30 191 ; 23 0 222 ; 5 59 99 ; 27 46 278 ; 11 58 84 ; 
36 53 104 ; 16 55 173 ; 10 44 404 ; 48 6 87 ; 9 41 478 ; 
40 0 83 ; 40 2 171 ; 35 6 238 ; 17 16 123 ; 12 40 227 ; 
1 28 218 ; 56 54 474 ; 36 50 90 ; 43 61 124 ; 
