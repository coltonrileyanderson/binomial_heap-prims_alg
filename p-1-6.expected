0: 47
1: 62(47)25
2: 40(62)47
3: 48(40)33
4: 22(48)18 31(48)26 53(48)45
5: 24(31)28 25(22)17 37(22)27 42(22)7
6: 3(24)6 7(42)15 13(42)9 27(42)10 29(25)10
7: 2(13)53 6(13)5 14(13)38 23(27)39 50(7)16 57(13)9 59(27)1
8: 8(23)11 11(50)29 30(59)50 32(6)90 36(50)38 38(59)30 39(59)8 44(23)24 49(57)0 52(23)63 56(6)34 60(50)13
9: 1(44)66 4(36)29 5(36)6 17(56)14 20(44)74 26(60)26 28(36)8 33(56)25 54(39)88 55(11)21
10: 0(5)46 9(33)53 21(33)5 34(28)12 35(54)44 41(28)25 45(33)8
11: 10(9)47 16(0)14 18(45)38 19(34)8 43(21)59 58(45)47 63(34)86
12: 12(18)11 15(58)67 46(10)46 61(19)32
13: 51(61)39
weight: 1918
